src/neg_tests/lin_tests_common_dsl.ml

```diff
@@ -61,8 +61,8 @@ struct
   let init () = CList.list_init 0
   let cleanup _ = ()
   let api =
-    [ val_ "CList.add_node" CList.add_node (t @-> int @-> returning bool);
-      val_ "CList.member"   CList.member  (t @-> int @-> returning bool);
+    [ val_ "CList.add_node" CList.add_node (t @-> nat_small @-> returning bool);
+      val_ "CList.member"   CList.member  (t @-> nat_small @-> returning bool);
```

Comment:
I think it is nice to keep type names (`int`, `int64`, ...) in the signatures to keep them as close to a genuine signature as possible. This is also the reason I defined `int` to something less uniform here:
  https://github.com/jmid/multicoretests/blob/main/src/hashtbl/lin_tests_dsl.ml#L12
Then we can later return and adjust/play with distributions by making changes in only one place.